### PR TITLE
LPS 153267

### PR DESCRIPTION
**Pull request by john-co**
- LPS-153267 Add options API for FDS views
- LPS-153267 Implement options API for FDS table view in FE
- LPS-153267 Configure quickActionsEnabled in options API in Sample FDS
- LPS-153267 Auto SF
- LPS-153267 Baseline
- LPS-153267 Quick actions on hover
- LPS-153267 Limit the number of visible quick actions
- LPS-153267 Add a new action in sample module
- LPS-153267 Auto SF
- LPS-153267 Prevent hover interactions on nested rows
- LPS-153267 Using context, not props, for quickActionsEnabled
- LPS-153267 Simplify hide-on-hover cell with pure CSS
- LPS-153267 Simplify using LinkOrButton Clay component
- LPS-153267 Cleanup
- LPS-153267 Rename
- LPS-153267 Have more than 3 dropdown items in FDS sample to test quick action menu

### CODE REVIEW

**Comment by john-co** [comment]
ci:test:sf

**Comment by john-co** [comment]
ci:test:sf

File: modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java [hunk 1]
```diff
@@ -62,11 +62,19 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 		return Arrays.asList(
 			new FDSActionDropdownItem(
 				null, "view", "sampleMessage",
-				LanguageUtil.get(_fdsRequestHelper.getRequest(), "view"), null,
+				LanguageUtil.get(_fdsRequestHelper.getRequest(), "Sample View"), null,
```
Comment (beltranrengifo) [inline modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java:65]: `Sample View` should be a lang key in portal lang, I'll add them to my [PR here](https://github.com/liferay-frontend/liferay-portal/pull/2262), ok?

File: modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java [hunk 2]
```diff
@@ -62,11 +62,19 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
...
 				null, null),
 			new FDSActionDropdownItem(
-				"#", "pencil", "sampleEditMessage",
-				LanguageUtil.get(_fdsRequestHelper.getRequest(), "edit"), null,
+				"#test-pencil", "pencil", "sampleEditMessage",
+				LanguageUtil.get(_fdsRequestHelper.getRequest(), "Sample Edit"), null,
+				null, null),
+			new FDSActionDropdownItem(
+				"#test-delete", "times-circle", "sampleDeleteMessage",
+				LanguageUtil.get(_fdsRequestHelper.getRequest(), "Sample Delete"), null,
```
Comment (beltranrengifo) [inline modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java:73]: Same

**Review by beltranrengifo — COMMENTED** [review]
Hi John! I'll add all 4 lang keys and translations in a commit in the main PR

Comment updated with the mentioned [commit](https://github.com/liferay-frontend/liferay-portal/pull/2262/commits/5e64213b8e62440b1049660b4411a2ed3fe42ebf)

File: modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java (hunk 1 above)
Comment (john-co) [reply modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java:65]: oh got it, thanks!

**Pull request closed without merging**